articles/architecture-scenarios/implementation/b2b/b2b-logout.md

```diff
@@ -14,10 +14,45 @@ useCase:
 
 <%= include('../../_includes/_logout/_introduction.md', { platform: 'b2b' }) %>
 
+## Single Logout
+
+If you are doing [Federated Logout](#federated-logout) you will likely also want to do Single Logout (SLO).
+
+::: warning
+SLO can add complexity to your system, so you need to ensure that you really need it before adding the extra development and maintenance time to your system.
+:::
+
+There are two main approaches you can take to Single Logout:
+
+### Short-lived tokens
+
+This is by far the simplest approach to Single Logout.  Each application enforces a short time that a user can use the system (5-10 minutes) and then on each action by that user if the time has expired, it will either redirect to Auth0 (for regular web apps), or use [Silent Authentication](https://auth0.com/docs/api-auth/tutorials/silent-authentication) for client side tokens.  This will ensure that eventually (within a minute or two) all other applications will fail to get their refreshed token because the SSO session has been removed through logout.
```

Comment:
```suggestion
This is by far the simplest approach to Single Logout. Each application enforces a short time within which a user can use the system, say, 5-10 minutes. On each action a user performs, if the time has expired then either a redirect to Auth0 (for regular web apps), or [Silent Authentication](https://auth0.com/docs/api-auth/tutorials/silent-authentication) for client side Single Page Applications will be used to obtain new tokens. Ordinarily new tokens will be issued silently due to the Single Sign On (SSO) session. However, after logout, all applications will fail to get new tokens silently because the SSO session will have been removed, and the user will need to re-enter their credentials.
```